.pipelines/mac-ios-spm-dev-validation-pipeline.yml

```diff
@@ -21,11 +21,17 @@
         cd tmp
         git clone -n --depth=1 --filter=tree:0 https://github.com/microsoft/onnxruntime.git
         cd onnxruntime
-        git sparse-checkout set --no-cone objectivec swift Package.swift
+        git sparse-checkout set --no-cone objectivec
         git checkout
       workingDirectory: "$(Build.SourcesDirectory)"
-      displayName: "Sparse checkout objectivec/ swift/ folders from latest ORT main repository"
-    
+      displayName: "Sparse checkout objectivec/ folders from latest ORT main repository"
```

Comment:
if all we need is the objectivec directory, maybe another approach would be to get it from pod-archive-onnxruntime-objc-*.zip

Reply:
Using the pod is fine for releases but wondering if there will be issues pre-release when testing changes. How will that work? Does it require a dev pod to be built/published somewhere and how will that work in a CI to validate the changes?